packages/asset-swapper/src/rpc_sampler_client.ts

```diff
@@ -0,0 +1,61 @@
+import axios from 'axios';
+
+import {
+    Address,
+    LiquidityResponse,
+    RpcLiquidityRequest,
+    TokenResponse,
+} from './utils/market_operation_utils/sampler_types';
+
+const SAMPLER_SERVICE_URL = '';
+
+export interface SamplerServiceInterface {
+    getSellLiquidityAsync(reqs: RpcLiquidityRequest[]): Promise<LiquidityResponse[]>;
+    getBuyLiquidityAsync(reqs: RpcLiquidityRequest[]): Promise<LiquidityResponse[]>;
+    getTokensAsync(tokenAddresses: Address[]): Promise<TokenResponse[]>;
+    // getPrices(tokenPaths: Address[][]): Promise<bigint[]>;
```

Comment:
This is now
```ts
get_prices(reqs: RpcPriceRequest[]): Promise<bigint[]>
```
where
```ts
interface RpcPriceRequest {
    tokenPath: Address[];
    sources?: LiquiditySource[];
    demand?: boolean;
}
```

Reply:
you should manually pass in `sources` and `demand: true` for now but in the future it might make sense to play with these for efficiency